aws/resource_aws_ec2_client_vpn_endpoint.go

```diff
@@ -304,11 +313,17 @@ func resourceAwsEc2ClientVpnEndpointUpdate(d *schema.ResourceData, meta interfac
 		}
 	}
 
-	_, err := conn.ModifyClientVpnEndpoint(req)
-	if err != nil {
+	if _, err := conn.ModifyClientVpnEndpoint(req); err != nil {
 		return fmt.Errorf("Error modifying Client VPN endpoint: %s", err)
 	}
 
+	if err := setTags(conn, d); err != nil {
+		return err
+	} else {
```

Comment:
Since the previous conditional calls `return`, the `else` and nesting are extraneous